src/resolvers/mutation.ts

```diff
@@ -35,10 +35,16 @@ export async function upsertSavedItem(
   const savedItemDataService = new SavedItemDataService(context);
 
   try {
-    const item = await ParserCaller.getOrCreateItem(savedItemUpsertInput.url);
+    //TODO do we need the resolved id @Herraj
```

Comment:
@kschelonka this came up during pairing. even though we are not exposing resolved_id through pocket graph's `savedItem`, we still populate resolved_id to maintain backward compatibility with web repo, right? 

otherwise, we were thinking to skip the parser call if the title is provided to prevent a downstream call